lib/ansible/plugins/action/eos.py

```diff
@@ -79,7 +79,7 @@ def run(self, tmp=None, task_vars=None):
                 # enable mode and not config module
                 rc, out, err = connection.exec_command('prompt()')
                 while str(out).strip().endswith(')#'):
-                    display.debug('wrong context, sending exit to device', self._play_context.remote_addr)
+                    display.debug('<%s> wrong context, sending exit to device' % self._play_context.remote_addr)
```

Comment:
I put it that way to match what display.verbose does, as I expect that is what we really want here, I'm just not sure at what level.